core/common/src/main/java/alluxio/underfs/ObjectUnderFileSystem.java

```diff
@@ -594,6 +598,21 @@ public boolean isObjectStorage() {
     return true;
   }
 
+  /**
+   * Checks if the root of the ObjectUnderFileSystem is listable. This is done once and cached
+   * per UFS to reduce time spent checking if the root is accessible.
+   *
+   * @return whether the root of the UFS is listable and return false if any exception is thrown
+   */
+  public boolean isRootListable() {
+    try {
+      return getObjectListingChunkForPath("", true) != null;
+    } catch (Exception e) {
+      LOG.debug("Unable to list bucket root:", e);
```

Comment:
Is there any more info you can add to this log message? Like the bucket name or anything else?

Reply:
done